syntax = "proto3";

option java_package = "com.grpc";

package helloworld;

service Greeter {
  rpc SayHello(HelloRequest) returns (HelloResponse) {
  }
}

message HelloRequest {
  string name = 1;
}

message HelloResponse {
  string message = 1;
}
